app/controllers/spree/admin/product_catalog_images_controller.rb

```diff
@@ -0,0 +1,48 @@
+module Spree
+  module Admin
+    class ProductCatalogImagesController < ResourceController
+      def update
+        begin
+          @variant.product_feed_image.destroy if @variant.product_feed_image
```

Comment:
why are we destroying the existing image if it exists? Something to do with deleting it in s3 or something akin to that?

Reply:
that's right, I was using that code to not upload unnecessary files, but I made this instead https://github.com/geminimvp/solidus_product_feed/pull/11/commits/f19498e2ac5834f375d76bdc1f19d6719aa8c735, please let me know what do you think